config/packages/smoketest/doctrine.yaml

```diff
@@ -3,10 +3,7 @@ doctrine:
     default_connection: gateway
     connections:
       gateway:
-        driver:   "%database_driver%"
-        port:     "%database_port%"
-        host:     db.stepup.example.com
-        dbname:   gateway
-        user:     gateway
-        password: gateway
-        charset:  UTF8
+        dbname: gateway_test
+        host: mariadb
+        password: gw_deploy_secret
+        user: gw_deploy_user
```

Comment:
@quartje I think you broke the database tests with this change (at least that's my hunch). Are you able to look at this, or should @parijke or me look into this?